airbyte-integrations/connectors/source-clockify/source_clockify/spec.yaml

```diff
@@ -4,8 +4,8 @@ connectionSpecification:
   title: Clockify Spec
   type: object
   required:
-  - workspace_id
-  - api_key
+    - workspace_id
```

Comment:
Does some new formatting stuff run automatically now? Curious since you didn't make any changes to these files.

Reply:
yeah, I think the format task now runs on the entire project? iirc this is new as of last week